### PR TITLE
added link to board meeting minutes on website

### LINKED ISSUE #195: Add link to Board meeting minutes on website
It would be nice to have a link to the [meeting minutes](https://github.com/sunpy/sunpy/wiki/Home%3A-Organization-Information#minutes-of-sunpy-board-meetings) on the [project page](https://sunpy.org/team.html#sunpy-board), to increase their visibility.

### PR DESCRIPTION
**Pull request by iamrajiv**
Fixes #195

### CODE REVIEW

**Comment by iamrajiv** [comment]
@nabobalis please take a look is it fine

**Comment by nabobalis** [comment]
You would also need to remove:

`CITATION.rst`
`exts/__pycache__/cards.cpython-37.pyc`
`exts/__pycache__/rawfiles.cpython-37.pyc`

otherwise it look good to me.

**Comment by iamrajiv** [comment]
@nabobalis means I have to delete the file before pushing

**Comment by nabobalis** [comment]
Yes.

**Comment by iamrajiv** [comment]
@nabobalis Is it good now

**Comment by nabobalis** [comment]
The citation file is still there.

**Comment by iamrajiv** [comment]
@nabobalis please take a look I think it's fine now

**Comment by nabobalis** [comment]
The citation file is still there?

**Comment by nabobalis** [comment]
It seems I can delete files from a PR? Github never ceases to amaze me.

File: team.rst [hunk 1]
```diff
@@ -269,3 +269,51 @@ Please send an email to the `mailing list`_ to start a dialogue.
 .. _Vishnunarayan K. I.: https://github.com/vn-ki
 .. _Will Barnes: https://github.com/wtbarnes
 .. _Bin Chen: https://github.com/binchensun
+
+Minutes of SunPy Board Meetings
+===============================
+
```
Comment (nabobalis) [inline team.rst]: I kind of feel this section needs some introduction?

**Comment by iamrajiv** [comment]
> It seems I can delete files from a PR? Github never ceases to amaze me.

@nabobalis yes you can I think it is because when I made this PR I had ticked `Allow edits from maintainers` option maybe that's why. Also, thank you for help and guidance

**Comment by nabobalis** [comment]
Thanks @iamrajiv

**Comment by dstansby** [comment]
Thanks!

**Pull request merged**